kebechet/managers/version/version.py

```diff
@@ -19,28 +19,42 @@
 
 import os
 import logging
+import typing
 
 from git import Repo
 from IGitt.Interfaces.Issue import Issue
 import yaml
+import semver
+from datetime import datetime
 
 from kebechet.utils import cloned_repo
 from kebechet.managers.manager import ManagerBase
 
 
 _LOGGER = logging.getLogger(__name__)
-_VERSION_REQUEST_ISSUE = ' release'
 _VERSION_PULL_REQUEST_NAME = 'Release of version {}'
-_NO_VERSION_FOUND_ISSUE_NAME = f"No version identifier found in sources to release "
-_MULTIPLE_VERSIONS_FOUND_ISSUE_NAME = f"Multiple version identifiers found in sources to release "
-_NO_MAINTAINERS_ERROR = "No release maintainers stated for this repository",
+_NO_VERSION_FOUND_ISSUE_NAME = f"No version identifier found in sources to perform a release"
+_MULTIPLE_VERSIONS_FOUND_ISSUE_NAME = f"Multiple version identifiers found in sources to perform a new release"
+_NO_MAINTAINERS_ERROR = "No release maintainers stated for this repository"
+_DIRECT_VERSION_TITLE = ' release'
+_RELEASE_TITLES = {
+    "New calendar release": lambda _: datetime.utcnow().strftime("%y.%m.%d"),
+    "New major release": semver.bump_major,
+    "New minor release": semver.bump_minor,
+    "New patch release": semver.bump_patch,
+    "New pre-release": semver.bump_prerelease,
+    "New build release": semver.bump_build,
+}
+
+
+class VersionError(Exception):
+    """An exception raised on invalid version provided or found in the repo."""
 
 
 class VersionManager(ManagerBase):
     """Automatic version management for Python projects."""
 
-    @staticmethod
```

Comment:
why is it not a static method anymore?

Reply:
Due to version computation in `_get_new_version`